format/SparseTensor.fbs

```diff
@@ -108,9 +108,22 @@ table SparseMatrixIndexCSR {
   indicesBuffer: Buffer;
 }
 
+/// Compressed Sparse Fiber (CSF) sparse tensor format
+///
+/// CSF is a generalization of compressed sparse row (CSR) index.
+/// CSF compresses a tensor into one three one-dimensional tensors.
+table SparseTensorIndexCSF {
+  indptrType: Int;
+  indptrBuffer: Buffer;
+  indicesType: Int;
+  indicesBuffer: Buffer;
+  axisOrder: [long];
```

Comment:
It would be nice to add a couple comments explaining the format, as done above for `SparseMatrixIndexCSR`.

Reply:
Will do.